src/main/java/randoop/condition/OperationConditions.java

```diff
@@ -0,0 +1,175 @@
+package randoop.condition;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Represents the collection of all conditions on an operation including pre-, post- and
```

Comment:
Actually, it does represent the collection of all preconditions, pre-post-condition pairs, and pre-throws-condition pairs.

I've changed the use of "condition" in the text to be more specific, but I'm leaving the class name, because I don't understand what you want it to be.